package-lock.json

```diff
@@ -2972,13 +2972,6 @@
             "dev": true,
             "license": "MIT"
         },
-        "node_modules/@tsd/typescript": {
-            "version": "4.8.4",
```

Comment:
This means `typescript@4.8.4` is currently used to test types. But the installed version of TypeScript is 5.4.5.

Hm.. Is this intentional? TSTyche can test agains specific version, e.g. `tstyche --target 4.8`. This command would use the latest version in `4.8` series (`4.8.4`). Or simply: `tstyche --target 4.8.4`.